common/proxy/proxy.go

```diff
@@ -140,6 +140,14 @@ func (p *Proxy) Implement(v common.RPCService) {
 				inv.SetAttachments(k, value)
 			}
 
+			// add user setAttachment
+			atm := invCtx.Value("attachment")
+			if m, ok := atm.(map[string]string); ok {
+				for k, value := range m {
```

Comment:
May null pointer error?

Reply:
when invCtx with no 'attachment',here ok is false